bindings/nodejs/examples/how_tos/nfts/mint_nft.ts

```diff
@@ -0,0 +1,129 @@
+
+// The owner address of the first NFT we'll mint
+const NFT1_OWNER_ADDRESS =
+    'rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu';
+// The metadata of the first minted NFT
+const NFT1_METADATA = utf8ToHex('some NFT metadata');
+// The immutable metadata of the first minted NFT
+const NFT1_IMMUTABLE_METADATA = utf8ToHex('some NFT immutable metadata');
+// The tag of the first minted NFT
+const NFT1_TAG = utf8ToHex('some NFT tag');
+// The base coin amount we sent with the second NFT
+const NFT2_AMOUNT = '1000000';
+
+// In this example we will mint a new nft.
+//
+// Make sure that `example.stronghold` and `example.walletdb` already exist by
+// running the `how_tos/accounts-and-addresses/create-wallet` example!
+//
+// Rename `.env.example` to `.env` first, then run
+// yarn run-example ./how_tos/nfts/mint_nft.ts
+async function run() {
+    try {
+        if (!process.env.STRONGHOLD_PASSWORD) {
+            throw new Error(
+                '.env STRONGHOLD_PASSWORD is undefined, see .env.example',
+            );
+        }
+
+        const wallet = new Wallet({
+            storagePath: process.env.WALLET_DB_PATH,
+        });
+
+        const account = await wallet.getAccount(
+            `${process.env.ACCOUNT_ALIAS_1}`,
+        );
+
+        // May want to ensure the account is synced before sending a transaction.
+        let balance = await account.sync();
+        const nftsBefore = balance.nfts;
+
+        // We send from the first address in the account.
+        const senderAddress = (await account.addresses())[0].address;
+
+        console.log('Sending the minting transaction for NFT 1...');
+
+        const params: MintNftParams = {
+            address: NFT1_OWNER_ADDRESS, // Remove or change to senderAddress to send to self
+            sender: senderAddress,
+            metadata: NFT1_METADATA,
+            tag: NFT1_TAG,
+            issuer: senderAddress,
+            immutableMetadata: NFT1_IMMUTABLE_METADATA,
+        };
+        const prepared = await account.prepareMintNfts([params]);
+
+        let transaction = await prepared.send();
+        console.log(`Transaction sent: ${transaction.transactionId}`);
+
+        // Wait for transaction to get included
+        let blockId = await account.retryTransactionUntilIncluded(
+            transaction.transactionId,
+        );
+
+        console.log(
+            `Transaction included: ${process.env.EXPLORER_URL}/block/${blockId}`,
+        );
+        console.log('Minted NFT 1');
+
+        // Build an NFT manually by using the `NftOutputBuilder`
+        const client = await wallet.getClient();
+
+        const hexAddress = Utils.bech32ToHex(senderAddress);
+        const output = await client.buildNftOutput({
+            amount: NFT2_AMOUNT,
+            nftId: '0x0000000000000000000000000000000000000000000000000000000000000000',
+            unlockConditions: [
+                new AddressUnlockCondition(new Ed25519Address(hexAddress)),
```

Comment:
This should be `Utils.bech32ToHex(NFT1_OWNER_ADDRESS)`?

Reply:
Yes. Changing it.

Reply:
Only in the AddressUnlockCondition it should be changed, the others should be as before